### PR TITLE
Improve docs of get_item

### PR DESCRIPTION
**Pull request by naoto0804**
This is PR for improving docs of functions/links. Related issue: #2182

Modified functions/links:
 - chainer/functions/array/get_item

### CODE REVIEW

**Comment by naoto0804** [comment]
screen_shot_get_item

File: chainer/functions/array/get_item.py [hunk 1]
```diff
@@ -68,12 +68,13 @@ def get_item(x, slices):
...
-        x (~chainer.Variable): A variable to be sliced.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): A variable to be sliced.
         slices (int, slice, Ellipsis, None, integer array-like, boolean\
         array-like or tuple of them):
             It is an integer, a slice, an ellipsis,
-            a numpy.newaxis, an integer array-like, a boolean array-like
-            or tuple of them.
+            a numpy.newaxis, a None, an integer array-like,
+            a boolean array-like or tuple of them.
```
Comment (niboshi) [inline chainer/functions/array/get_item.py]: I think this whole sentence is redundant. How about more general expression, such as "an object to specify the selection of elements"?

Comment (naoto0804) [reply chainer/functions/array/get_item.py]: Sounds much better, thank you!

**Comment by niboshi** [comment]
Could you fix it?
I think I can merge as soon as it's fixed.

**Comment by niboshi** [comment]
@naoto0804
Can you work on this PR? If you can't afford time, we can take over this PR.
Thank you very much!

**Comment by naoto0804** [comment]
@niboshi I'm sorry for the late reply, could you kindly take over this PR?

**Comment by niboshi** [comment]
@naoto0804
Sure.
Thank you very much for your contribution on this PR!

**Comment by niboshi** [comment]
I opened #4375, so I'm closing this PR.

**Pull request closed without merging**